src/components/networked.js

```diff
@@ -302,16 +323,89 @@ AFRAME.registerComponent('networked', {
               childEl.setAttribute(schema.component, schema.property, data);
             }
             else {
-              childEl.setAttribute(schema.component, data);
+              this.updateComponent(childEl, schema.component, data);
             }
           }
         } else {
-          el.setAttribute(key, data);
+          this.updateComponent(el, key, data);
         }
       }
     }
   },
 
+  updateComponent: function (el, key, data) {
+    switch(key) {
+      case "position":
+        var posComp = this.positionComponents.find((item) => item.el === el);
+
+        if (!posComp) {
+          posComp = {};
+          posComp.el = el;
+          posComp.start = new THREE.Vector3(data.x, data.y, data.z);
+          posComp.target = new THREE.Vector3(data.x, data.y, data.z);
+          posComp.lastUpdated = Date.now();
+          posComp.duration = 1;
+          this.positionComponents.push(posComp);
+        } else {
+          posComp.start.copy(posComp.target);
```

Comment:
This is the same lerping strategy that `aframe-lerp-component` uses but is only valid for data that is continously being updated. For objects that aren't continously moving this results in some pretty undesirable behavior. Since this isn't a regression it's fine for this PR but we definitely need to think of ways to allow other interpolation strategies.